wavefront-hpa-adapter/templates/custom-metrics-apiserver-deployment.yaml

```diff
@@ -21,10 +21,22 @@ spec:
       - name: custom-metrics-apiserver
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+      {{- if .Values.wavefront.secret }}
+        env:
+        - name: WAVEFRONT_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.wavefront.secret.name }}
+              key: {{ .Values.wavefront.secret.key }}
```

Comment:
Could we assume the secret.key to be consistent i.e. hardcoded to "token" or do you have a specific use case to configure it ?
It will reduce one more avenue of misconfiguring something and setting it to "token" will help stay consistent with what our [observability-for-kubernetes](https://github.com/wavefrontHQ/observability-for-kubernetes#deploy-the-monitoring-agents-with-the-observability-for-kubernetes-operator) operator uses. (See step 2)

Reply:
Done @akodali18 ! In addition to this, I raised a PR about this in the https://github.com/wavefrontHQ/wavefront-kubernetes-adapter repository, I don't know if [this](https://github.com/wavefrontHQ/wavefront-kubernetes-adapter/pull/47/files) could help to use the option to set the environment variables to not use the binary flags.